labgrid/driver/lauterbachdriver.py

```diff
@@ -0,0 +1,143 @@
+import logging
+import os
+import re
+import subprocess
+
+from importlib import import_module
+
+import attr
+
+from ..factory import target_factory
+from ..protocol import DebuggerProtocol
+from ..resource.lauterbach import NetworkLauterbachDebugger, NetworkUSBLauterbachDebugger
+from ..resource.udev import USBLauterbachDebugger
+from ..util.helper import get_uname_machine, processwrapper
+from ..util.managedfile import ManagedFile
+from ..util.proxy import proxymanager
+from ..util import Timeout
+
+from .common import Driver
+
+
+@target_factory.reg_driver
+@attr.s(eq=False)
+class LauterbachDriver(Driver, DebuggerProtocol):
+    """
+    Args:
+        t32_sys (str): base folder of the TRACE32 installation (default ENV['T32SYS'])
+        t32_bin (str): name of the TRACE32 architecture executable `t32m*` (default `t32marm`)
+    """
+    bindings = {
+        "interface": {
+            "USBLauterbachDebugger",
+            "NetworkUSBLauterbachDebugger",
+            "NetworkLauterbachDebugger"
+        },
+    }
+    t32_bin = attr.ib(default="t32marm", validator=attr.validators.instance_of(str))
```

Comment:
Maybe have a look at the [QEMUDriver's](https://labgrid.readthedocs.io/en/latest/configuration.html#qemudriver) `qemu_bin` attribute.